gcc/rust/ast/rust-expr.h

```diff
@@ -5,6 +5,7 @@
 #include "rust-path.h"
 #include "rust-macro.h"
 #include "rust-operators.h"
+#include <memory>
```

Comment:
this system include should be replaced by `#include "rust-system.h"` which should provide the content of `<memory>`
```suggestion
#include "rust-system.h"
```

Reply:
resolved